src/dialogs/more-info/controls/more-info-weather.ts

```diff
@@ -203,6 +203,149 @@ class MoreInfoWeather extends LitElement {
             )}
           `
         : ""}
+      ${this.stateObj.attributes.forecast_daily
```

Comment:
We should not show all forecasts in the more info, that is way too much data

Reply:
Discussed it shortly with @emontnemery that we should set some priority and not show all of them so it will be adjusted.

Reply:
Can we have a default priority? Show the most precise one? Or combine them into 1?